apis/go.mod

```diff
@@ -3,8 +3,11 @@ module github.com/metal3-io/baremetal-operator/apis
 go 1.16
 
 require (
+	github.com/metal3-io/baremetal-operator/pkg/ironic v0.0.0-20211108163011-2e5b1ff824a2
```

Comment:
This module can not be imported in here because there is circular dependeny.

https://github.com/metal3-io/baremetal-operator/pull/1017 will fix it.